src/gui/static/src/app/components/pages/wallets/create-wallet/create-wallet.component.ts

```diff
@@ -34,13 +34,14 @@ export class CreateWalletComponent implements OnInit {
   }
 
   createWallet() {
+    this.createButton.resetState();
     this.createButton.setLoading();
     this.cancelButton.setDisabled();
     this.disableDismiss = true;
 
     const password = this.encrypt ? this.form.value.password : null;
     this.walletService.create(this.form.value.label, this.form.value.seed, this.scan, password)
-      .subscribe(() => this.dialogRef.close());
+      .subscribe(() => this.dialogRef.close(), e => this.createButton.setError(e));
```

Comment:
It is necessary to add the following lines in the error handler, for making possible to close the dialog after an error:
```
this.cancelButton.disabled = false;
this.disableDismiss = false;
```

Reply:
@Senyoret1 can you add it? then we can merge it